src/main/java/org/zalando/nakadi/plugin/api/authz/AuthorizationService.java

```diff
@@ -22,6 +23,18 @@
      */
     boolean isAuthorized(Operation operation, Resource resource) throws PluginException;
 
+    /**
+     * Check whether a Resource is valid.
+     *
+     * Example: Take an attribute with key 'username' and value 'nakadi'. A plugin implementing this method could
+     * check that (a) 'username' is an accepted key, and (b) 'nakadi' is a username that exists and is active.
```

Comment:
I'm not sure id example description is correct, I think it describes a case for `boolean isAuthorizationAttributeValid(AuthorizationAttribute attribute)` method.